saleor/graphql/webhook/subscription_payload.py

```diff
@@ -54,10 +55,11 @@ def initialize_context() -> HttpRequest:
     return: HttpRequest
     """
     handler = BaseHandler()
-    context = RequestFactory().request()
+    context = RequestFactory().request(**{"SERVER_NAME": settings.ALLOWED_HOSTS[-1]})
```

Comment:
Why -1? See how we [set up this value for WSGI app](https://github.com/saleor/saleor/blob/ebc0e0e66168c358373d594c0c50865ef8e6507f/saleor/wsgi/__init__.py#L37), maybe we should reuse it here.